jupyter_server/services/contents/manager.py

```diff
@@ -126,7 +126,6 @@ def _notary_default(self):
             "*.pyc",
             "*.pyo",
             ".DS_Store",
-            "*.so",
             "*.dylib",
```

Comment:
```suggestion
```

Suggesting this change as per https://github.com/jupyter-server/jupyter_server/pull/1457#issuecomment-2631748439